tests/acceptance/pageObjects/phoenixPage.js

```diff
@@ -116,6 +116,13 @@ module.exports = {
           .click(declineShareButton)
           .waitForAjaxCallsToStartAndFinish()
       }
+    },
+    getPhoenixcontainer: async function () {
```

Comment:
Naming is wrong. It returns boolean, not the container.

Reply:
Would you mind recommending some?